src/hb-font.cc

```diff
@@ -2442,6 +2451,74 @@ hb_font_get_ptem (hb_font_t *font)
   return font->ptem;
 }
 
+/**
+ * hb_font_set_synthetic_bold:
+ * @font: #hb_font_t to work upon
+ * @x_embolden: the amount to embolden horizontally
+ * @y_embolden: the amount to embolden vertically
+ * @in_place: whether to embolden glyphs in-place
+ *
+ * Sets the "synthetic boldness" of a font.
+ *
+ * Positive values for @x_embolden / @y_embolden make a font
+ * bolder, negative values thinner. Typical values are in the
+ * 0.01 to 0.05 range. The default value is zero.
+ *
+ * Synthetic boldness is applied by offsetting the contour
+ * points of the glyph shape.
+ *
+ * Synthetic boldness is applied when rendering a glyph via
+ * hb_font_draw_glyph().
+ *
+ * If @in_place is %FALSE, then glyph advance-widths are also
+ * adjusted, otherwise they are not.
```

Comment:
It might be nice to give a hint about the use case for @in_place here.

Reply:
I'll mention grading here and link to https://fonts.google.com/knowledge/glossary/grade

Reply:
@khaledhosny What's the docbook syntax for links? https://developer-old.gnome.org/gtk-doc-manual/stable/documenting_syntax.html.en only has the markdown syntax. Is that kosher in our docs?

Reply:
> Is that kosher in our docs?

Seems to work.